_inc/client/traffic/search.jsx

```diff
@@ -39,7 +39,7 @@ class Search extends React.Component {
 					{ this.props.getOptionValue( 'search' ) && (
 						<FormFieldset>
 							<p className="jp-form-setting-explanation">
-								{ __( 'To configure search filters add the {{link}}Jetpack Search widget to your sidebar{{/link}}.', {
+								{ __( 'To configure search filters add the {{link}}"Search (Jetpack)" widget to your sidebar{{/link}}.', {
```

Comment:
I'm sorry, but we can't change strings now. I should have mentioned it before, but the strings that went out with the code freeze have already been sent to the community translation portal and most of them have been already translated in most popular languages. If we change it now, this string will appear untranslated until we ship another release.